src/config/mongoose_config_parser_toml.erl

```diff
@@ -0,0 +1,354 @@
+-module(mongoose_config_parser_toml).
```

Comment:
Should all the options be parsed in a single file? It seems like it could become monstrous in the future

Reply:
I fully agree and they are currently in one place to simplify the first implementation. Apart from splitting it is quite likely that a callback mechanism will be used.